### ISSUE TITLE
[java] DefaultPackage is referencing incorrect ParameterizedTest

### ISSUE THREAD

**Issue by Oleg-Pavlenko-EPAM** [pmd/pmd#3301]
[java]

**Affects PMD Version:** 

`6.29.0`, the latest compatible with maven plugin `3.14.0` but according to the documentation it is still present in the latest version.

**Rule:**

<https://pmd.github.io/pmd-6.29.0/pmd_rules_java_codestyle.html#defaultpackage>

**Description:**

The rule has the exclusion for `org.junit.jupiter.api.ParameterizedTest` which actually doesn't exist. The correct one is `org.junit.jupiter.params.ParameterizedTest`. See https://junit.org/junit5/docs/5.0.2/api/org/junit/jupiter/params/ParameterizedTest.html

**Code Sample demonstrating the issue:**

```
    @ParameterizedTest
    @MethodSource(...)
    void testSomething(String parameter1, String parameter2) {
        ...
    }
```

**Expected outcome:**

PMD reports a violation at line 1, but that's wrong. That's a false positive.

**Running PMD through:** *Maven*

**Comment by oowekyala** [comment]
The rule is deprecated from 6.35.0 on (#3206). This won't be fixed.
